web/views.py

```
@@ -6929,3 +6947,449 @@ def delete_post(request, post_id):
+        # Record event
+        MembershipSubscriptionEvent.objects.create(
+            user=user,
+            membership=membership,
+            event_type="canceled",
+            stripe_event_id=subscription.get("id", ""),
+            data={"subscription_id": subscription.get("id", ""), "canceled_at": timezone.now().isoformat()},
+        )
+
+        logger.info("Subscription deleted for user %s via webhook", user.email)
+    except User.DoesNotExist:
+        logger.error("No user found for customer %s", subscription["customer"])
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)